docs/book/user-guide/starter-guide/connect-to-a-deployed-zenml.md

```diff
@@ -36,16 +59,24 @@ Password for user zenml (press ENTER for empty password) []:
 ```
 
 {% hint style="info" %}
-In Scenario 2, the `zenml up` command implicitly connects the client to the server.
+In Scenario 2, the `zenml up` command implicitly connects the client to the
```

Comment:
What do you think of moving this info block below the Scenario 2 figure?

Reply:
done